Rework zh l10n; correct info in en msg; make "show help" localizable

pzhlkj6612 opened the pull request:
## Fixes no issue

### Changes Proposed in this Pull Request:

- Correct some information in English messages.
- Rework the Simplified Chinese L10N.
- Localize the "Show Help" button in the context menu 

### Additional Comments and Documentation:

The Chinese content and English one in the message files, in this PR, are not one-to-one corresponding. The main issue of the original Zh L10N is that it's too heavily translated, so too hard to understand. Please don't make me wrong as I'm not blaming anyone. Translation is tough. Everyone will suffer from headaches when choosing words and phrases. To make all Chinese messages more clear, I have to use "paraphrase" instead of "direct translation". That's the story.

pzhlkj6612 committed:
Rework Zh l10n; correct info in En msg; localize "Show Help" button

brandon1024 commented:
@pzhlkj6612 could you rebase this onto the tip of `develop`?

```
$ git fetch --all
$ git rebase --onto origin/develop HEAD^ fix/l10n-zh
```

pzhlkj6612 commented:
Weird. Does Firefox have no way to display the "Show Help" button anywhere? So how do Firefox users read the documentation?

pzhlkj6612 committed:
Fix broken menu of browser action on Firefox

Firefox is still MV2: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/menus/ContextType#browser_action

... banged my head against the wall!

pzhlkj6612 commented:
> Weird. Does Firefox have no way to display the "Show Help" button anywhere? So how do Firefox users read the documentation?

Okay, fixed in e2dc6088d1bd7a8a019bc764e80aa1bb5215cf8c. I don't know that we are still using MV2 Firefox extension.

Upgrading to MV3 for Firefox has been done in https://github.com/brandon1024/find/pull/399#issuecomment-2817195291. I don't think there is anything blocking us to do so.

brandon1024 commented:
LGTM!

pzhlkj6612 commented:
Thank you! And, 

> Upgrading to MV3 for Firefox has been done in [#399 (comment)](https://github.com/brandon1024/find/pull/399#issuecomment-2817195291). I don't think there is anything blocking us to do so.

Any opinion about this?
